packages/elements/src/header/index.ts

```diff
@@ -56,6 +59,13 @@ export class Header extends BasicElement {
   @property({ type: String, reflect: true })
   public level: '1'| '2'| '3' = '2';
 
+  protected updated (changedProperties: PropertyValues): void {
+    super.updated(changedProperties);
+    if (changedProperties.has('level')) {
+      this.setAttribute('aria-level', this.level);
```

Comment:
I would suggest to define `ariaLevel` as reflect property and change `level` to setter/getter. We usually use this pattern in the others element e.g. `checkbox`.

Reply:
We decided to developer handle attribute `role` and `aria-level` at the application level so the element header does not need to change.